tests/scripts/github_mergebot.py

```diff
@@ -33,7 +34,7 @@
 CIJob = Dict[str, Any]
 
 EXPECTED_JOBS = ["tvm-ci/pr-head"]
-THANKS_MESSAGE = "Thanks for contributing to TVM!   Please refer to guideline https://tvm.apache.org/docs/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/apache/incubator-tvm/blob/master/CONTRIBUTORS.md#reviewers) by @ them in the pull request thread."
+THANKS_MESSAGE = r"(\s*)Thanks for contributing to TVM!   Please refer to guideline https://tvm.apache.org/docs/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from \[Reviewers\]\(https://github.com/apache/incubator-tvm/blob/master/CONTRIBUTORS.md#reviewers\) by  them in the pull request thread.(\s*)"
```

Comment:
should we just pull from https://github.com/apache/tvm/blob/main/.github/PULL_REQUEST_TEMPLATE.md?

Reply:
We could but idk if it's worth the extra network request for something that hasn't been changed in over a year